src/core/rowindex_slice.cc

```diff
@@ -41,8 +41,8 @@ bool check_slice_triple(size_t start, size_t count, size_t step, size_t max)
   return (start <= max) &&
          (count <= RowIndex::MAX) &&
          (count <= 1 || (step == 0) ||
-          (step <= RowIndex::MAX? step <= (max - start)/(count - 1)
-                                : step >= -start/(count - 1)));
+         (step <= RowIndex::MAX? step <= (max - start)/(count - 1)
+                               : step >= -start/(count - 1)));
```

Comment:
I thought we decided that `-step <= start/(count - 1)` is better?

Reply:
Yeah, but at some point I stopped fixing unsigned integer issues as we have quite a lot of them and they all need special attention (somewhere we may even have a bug). So will address all them in the subsequent PR.